### PR TITLE
QAT readme edits for format and text flow.

### PR DESCRIPTION
**Pull request by MCamp859**
@swatisehgal  Please review these text edits. I have a couple open questions for you. 

Signed-off-by: MCamp859 <mary.camp@ptiglobal.net>

### CODE REVIEW

File: cmd/qat_plugin/README.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
-  * [Linux Drivers](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
-* Ensure that the QuickAssist SR-IOV virtual functions are configured. Verify this by running:
+### Prerequisites
+* Data Plane Development Kit (DPDK) drivers must be loaded and configured. For more information, refer to:
+    * [DPDK Getting Started Guide for Linux](https://doc.dpdk.org/guides/linux_gsg/index.html)
+    * [DPDK Getting Started Guide, Linux Drivers section](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
+* QuickAssist SR-IOV virtual functions must be configured. Verify this by running:
 ```
 lspci | grep QAT
 ```
```
Comment (MCamp859) [inline cmd/qat_plugin/README.md]: AR Mary fix indent on lines 8-10.

File: cmd/qat_plugin/README.md [hunk 2]
```diff
@@ -71,29 +82,31 @@ $ kubectl describe node <node name> | grep intel.com/qat
...
-   ```
-   $ kubectl create -f demo/crypto-perf-dpdk-pod-requesting-qat.yaml
-   $ kubectl get pods
-     NAME                     READY     STATUS    RESTARTS   AGE
-     dpdkqatuio               1/1       Running   0          27m
-     intel-qat-plugin-5zgvb   1/1       Running   0          3h
+      In the pod specification file, add container resource request and limit.
+      For example, `intel.com/qat: <number of devices>` for a container requesting QAT devices.
 
+      For a DPDK based workload, you may need to add hugepage request and limit.
```
Comment (MCamp859) [inline cmd/qat_plugin/README.md]: @swatisehgal  Please see if step 2 makes sense - should the user first update the pod spec file and then run the example DPDK application? (I am wondering if lines 94-97 are step 2, then step 2 becomes 3, etc.)

File: cmd/qat_plugin/README.md [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
 $ ls /var/lib/kubelet/device-plugins/kubelet.sock
 /var/lib/kubelet/device-plugins/kubelet.sock
 ```
-### Build QAT device plugin
+
+### Build QAT device plugin:
 ```
 $ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
 $ make qat_plugin
 ```
```
Comment (MCamp859) [inline cmd/qat_plugin/README.md]: @swatisehgal  The other plugin readmes have this order: Get source code, Build plugin, Verify kubelet socket, etc.
Is it ok to switch these 2 steps (Verify kubelet socket and Build QAT device plugin)?

File: cmd/qat_plugin/README.md (hunk 2 above)
Comment (swatisehgal) [reply cmd/qat_plugin/README.md]: yes that's correct the pod specification needs to be updated before running the DPDK application. May be we can change this as follows:
2. Deploy a Pod to run the DPDK application (dpdk-test-crypto-perf)
In the pod specification file, add container resource request and limit. For example, intel.com/qat: <number of devices> for a container requesting QAT devices.

For a DPDK based workload, you may need to add hugepage request and limit.

$ kubectl create -f demo/crypto-perf-dpdk-pod-requesting-qat.yaml
$ kubectl get pods
  NAME                     READY     STATUS    RESTARTS   AGE
  dpdkqatuio               1/1       Running   0          27m
  intel-qat-plugin-5zgvb   1/1       Running   0          3h

3. Manually execute the dpdk-test-crypto-perf application to review the logs. 
$ kubectl exec -it dpdkqatuio bash
$ ./dpdk-test-crypto-perf -l 6-7 -w $intelQAT1 -- --ptest throughput --\
 devtype crypto_qat --optype cipher-only --cipher-algo aes-cbc --cipher-op \
 encrypt --cipher-key-sz 16 --total-ops 10000000 --burst-sz 32 --buffer-sz 64

Comment (swatisehgal) [reply cmd/qat_plugin/README.md]: I hope that makes sense, let me know if you have questions :)

File: cmd/qat_plugin/README.md (hunk 3 above)
Comment (swatisehgal) [reply cmd/qat_plugin/README.md]: Yes, no problem, we can be consistent with other plugins.

Comment (MCamp859) [reply cmd/qat_plugin/README.md]: @swatisehgal   Done, please review the latest file.

File: cmd/qat_plugin/README.md (hunk 2 above)
Comment (MCamp859) [reply cmd/qat_plugin/README.md]: @swatisehgal  Done, please review the latest.

File: cmd/qat_plugin/README.md [hunk 4]
```diff
@@ -89,7 +89,8 @@ $ kubectl describe node <node name> | grep intel.com/qat
 
      This command produces a Docker image named `ubuntu-demo-opencl`.
```
Comment (swatisehgal) [inline cmd/qat_plugin/README.md]: We have a typo which I missed to fix. Can you please change this to:
This command produces a Docker image named crypto-perf

Comment (MCamp859) [reply cmd/qat_plugin/README.md]: Done!

**Comment by MCamp859** [comment]
@rojkov @bart0sh  Ready for your approval & merge, thanks.

**Pull request merged**